XamarinCommunityToolkit/Views/MediaElement/UriTypeConverter.shared.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using Xamarin.Forms;
+using Xamarin.Forms.Xaml;
+
+namespace Xamarin.CommunityToolkit.UI.Views
+{
+	[TypeConversion(typeof(Uri))]
+	public class UriTypeConverter : TypeConverter
+	{
+		public override object ConvertFromInvariantString(string value)
+		{
+			if (string.IsNullOrWhiteSpace(value))
+				return null;
+			return new Uri(value, UriKind.RelativeOrAbsolute);
+		}
+
+		bool CanConvert(Type type)
```

Comment:
@jsuarezruiz will we use this method or can I remove it?

Reply:
Is not used, we can remove it.